README.md

```diff
@@ -1 +1,11 @@
-# angular-routing-testing-lab
\ No newline at end of file
+# Writing tests for routing
+
+## Objectives
+
+- Integrate unit tests for our routing configurations
+
+## Instructions
+
+We've got our three states setup in this repo - settings, user settings and notification settings.
+
+Using the knowledge from our previous README, add our tests to ensure that all of our URLs are correct, that we're using the correct controller and that the templateUrl is correct.
```

Comment:
Here in the instructions, let's let the user know what the values should be for the URL and the templateUrl at each of the different states, that way it'll be easier to implement the tests.